learn/app-development/services/java-services/java-integration-database-service.md

```diff
@@ -0,0 +1,191 @@
+---
+title: "Integrate Database Service in JavaServices"
+id: ""
+---
+---
+
+We can invoke Database Service from Java by `@Autowired` the service files. 
```

Comment:
Can you start with explaining why someone would want to do this? @SaraswathiRekhala

Reply:
Added more info